realsense2_camera/src/rs_node_setup.cpp

```diff
@@ -152,14 +161,16 @@ void BaseRealSenseNode::setAvailableSensors()
 
     std::function<void()> hardware_reset_func = [this](){hardwareResetRequest();};
 
-    _dev_sensors = _dev.query_sensors();
-
     for(auto&& sensor : _dev_sensors)
     {
         const std::string module_name(rs2_to_ros(sensor.get_info(RS2_CAMERA_INFO_NAME)));
         std::unique_ptr<RosSensor> rosSensor;
-        if (sensor.is<rs2::depth_sensor>() ||
-            sensor.is<rs2::color_sensor>())
+        if (sensor.is<rs2::depth_sensor>())
```

Comment:
If and else code match, what was the purpose here?

Reply:
right